packages/world/test/SystemHook.t.sol

```diff
@@ -45,4 +45,8 @@ contract SystemHookTest is Test, GasReporter {
     assertEq(systemHook.getAddress(), hookAddress);
     assertEq(systemHook.getBitmap(), enabledHooksBitmap);
   }
+
+  function testShorthand() public {
+    assertEq(ALL, BEFORE_CALL_SYSTEM | AFTER_CALL_SYSTEM);
+  }
```

Comment:
is this a useful test? 😆

Reply:
haha just want to make sure `ALL` is updated in case we add more hooks